lib/index.js

```diff
@@ -19,6 +19,20 @@ var render = require('posthtml-render')
  */
 function PostHTML (plugins) {
   this.plugins = plugins || []
+
+  /**
+   * @example
```

Comment:
```js
this.options = {}
```
it is not necessary as [determined in the process](https://github.com/GitScrum/posthtml/blob/master/lib/index.js#L74-L91)